src/plugins/summaryWidget/src/telemetry/SummaryWidgetMetadataProvider.js

```diff
@@ -0,0 +1,95 @@
+define([
+
+], function (
+
+) {
+
+    function SummaryWidgetMetadataProvider(openmct) {
+        this.openmct = openmct;
+    }
+
+    SummaryWidgetMetadataProvider.prototype.appliesTo = function (domainObject) {
+        return domainObject.type === 'summary-widget';
+    };
+
+    SummaryWidgetMetadataProvider.prototype.getDomains = function (domainObject) {
```

Comment:
Assuming this is not part of the intended interface for metadata providers?

Reply:
Correct.  internal usage only.